src/main/java/seedu/address/model/person/UniquePersonList.java

```diff
@@ -103,6 +109,118 @@ public void setPersons(List<? extends ReadOnlyPerson> persons) throws DuplicateP
         return FXCollections.unmodifiableObservableList(mappedList);
     }
 
+    /**
+     * Sorts the list by field.
+     */
+    public void sort(String field) {
```

Comment:
Can this be more abstractive?
Basically can we put the comparators in the respective classes.
There seems to be a lot of repetitive code here.
Just a suggestion, perhaps leave a TODO mark here for future improvement?